README.md

```diff
@@ -1,13 +1,38 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
-
-## Available Scripts
-
-In the project directory, you can run:
-
-### `npm start`
-
-Runs the app in the development mode.<br />
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+# Horondi
+Frontend app for the `Hondri` online store
+## Rules and guidelines
+- Redux
+    - For each entity we should have separate folder
+    - In each folder we should have different files for actions, reducer
+      `{modelName}.actions.js` or `{modelName}.reducer.js`
+- Configuration
+    - Configuration is done via `.env` file where environment
+      variables are located
+    - Also we have `.env.example` that contains examples of environment
+      variables
+- Styles
+    - For styling function `makeStyles` from `@material-ui`
+      should be used and all styles should be located inside separate
+      component.
+- Components
+    - Components that are connected to redux should be located inside
+      `containers` folder. Components without connection to redux should
+      be located inside `components` folder.
+    - Each individual page that is accessed via `react-router`
+      should be located inside `pages` folder. All components
+      that are used inside particular page should be located inside
+      folder for the specific page.
+    - Each component should have at least three files:
+      - `index.js` where we export anything from the whole folder
+      - `{component-name}.js` - file where component is located
+      - `{component-name}.styles.js` where all styles are located
+
+## Starting a project
+In the project directory, you should run:
+- Copy content of `.env.example` file to the newly created `.env`
+- Run `npm install` to install all dependencies
+- Run `npm run start` to start application
```

Comment:
The application will start only if the backend is deployed already. 
Otherwise, you should run the backend from https://github.com/horondi/horondi_client_be